views/templates/admin/hookbackoffice.tpl

```diff
@@ -24,305 +24,344 @@
 *  International Registered Trademark & Property of PrestaShop SA
 *}
 
-<style>
-.isa_info, .isa_success, .isa_warning, .isa_error {
-	margin: 10px 0px;
-	padding:12px;
-	border-radius:5px;
-}
-.isa_info {
-	color: #00529B;
-	background-color: #BDE5F8;
-}
-.isa_success {
-	color: #4F8A10;
-	background-color: #DFF2BF;
-}
-.isa_warning {
-	color: #9F6000;
-	background-color: #FEEFB3;
-}
-.isa_error {
-	color: #D8000C;
-	background-color: #FFBABA;
-}
-.isa_info i, .isa_success i, .isa_warning i, .isa_error i {
-	margin:10px 22px;
-	font-size:2em;
-	vertical-align:middle;
-}
-</style>
+<link href="{$content_data['module_dir']|escape:'htmlall':'UTF-8'}/views/css/sellstrom.css" rel="stylesheet" type="text/css">
+<script type="text/javascript" src="{$content_data['module_dir']|escape:'htmlall':'UTF-8'}/views/js/sellstrom.js"></script>
```

Comment:
Do not include your files this way. Please use instead the functions addCss() and addJs(). Here is an example here:
https://github.com/PrestaShop/gadwords/blob/83d13209fb71bafab9307839cb27c78317dade78/gadwords.php#L66